pom.xml

```diff
@@ -389,6 +389,16 @@
           <artifactId>properties-maven-plugin</artifactId>
           <version>1.0.0</version>
         </plugin>
+        <plugin>
+          <groupId>io.quarkus</groupId>
+          <artifactId>quarkus-maven-plugin</artifactId>
+          <version>1.6.1.Final</version>
+        </plugin>
+        <plugin>
+          <groupId>org.jboss.jandex</groupId>
+          <artifactId>jandex-maven-plugin</artifactId>
+          <version>1.0.7</version>
```

Comment:
you could even add the `<execution>` configuration here as it is the same for all modules, right?